src/test/java/uk/gov/pay/adminusers/resources/ServiceResourceUpdateMerchantDetailsTest.java

```diff
@@ -82,6 +86,7 @@ public void shouldFail_whenUpdatingMerchantDetails_withoutMandatoryFields() thro
         Service service = Service.from(randomInt(), serviceExternalId, "existing-name");
         databaseHelper.addService(service, randomInt().toString());
         Map<String, Object> payload = ImmutableMap.of(
+                "telephone_number", "07700900000",
```

Comment:
I know it was like this when you got here but this test would be better if it tested what happens if you miss just one mandatory field — right now it tests what happens when you miss all of them, which doesn’t feel as useful